tests/Integration/AuthenticationTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace FtwSoft\Rundeck\Tests\Integration;
+
+use FtwSoft\Rundeck\Authentication\PasswordAuthentication;
+use FtwSoft\Rundeck\Authentication\TokenAuthentication;
+use FtwSoft\Rundeck\Client;
+use GuzzleHttp\Client as HttpClient;
+use Http\Factory\Guzzle\RequestFactory;
+use Http\Factory\Guzzle\StreamFactory;
+use PHPUnit\Framework\TestCase;
+
+class AuthenticationTest extends TestCase
+{
+    public function testPasswordAuthentication()
+    {
+        $httpClient = new HttpClient();
+
+        $authentication = new PasswordAuthentication(
+            getenv('RUNDECK_HOST'),
```

Comment:
It's better to use setUp method to receive values from ENV into class properties

Reply:
Agreed.